### PR TITLE
Port HoverCard animation fix to 5.0

### PR DESCRIPTION
**Pull request by ThomasMichon**
Bringing over a bugfix for an animation issue in `HoverCard` impacting IE 11.
Also adding `lib-commonjs` to `.gitignore` to make it easier to switch between branches.

### CODE REVIEW

**Comment by ThomasMichon** [comment]
@Jahnp and @atneik, we need this to resolve a support issue. If you don't get back by tomorrow I'll override!

**Comment by Jahnp** [comment]
@ThomasMichon ack--this was already done in master right?

**Comment by atneik** [comment]
@Jahnp Yes I got know yesterday that it is fixed in v6, but not in v5

File: .gitignore [hunk 1]
```diff
@@ -43,6 +43,7 @@ temp
 lib
 lib-amd
 lib-es2015
+lib-commonjs
```
Comment (cliffkoh) [inline .gitignore]: :+1:

**Pull request merged**